### ISSUE TITLE
OpenSSL seems to not to be recommended any more

### ISSUE THREAD

**Issue by blaggacao** [nubark/facturark#11]
Unrecommended security package

Please see how crypto is imported and used:
https://github.com/odoo/enterprise/blob/4842c43917b135a1c4fb637fba5a653b4ece5222/l10n_mx_edi/models/certificate.py#L12-L15

Unfortunately, this is contrary to the recommendation of the package authors:
https://pypi.org/project/pyOpenSSL/ (first paragraph).

It says "strongly suggests".

Also, https://github.com/pyca/cryptography is likely to be(come) more widely used.

**Comment by tebanep** [comment]
That is true, but it was the only way we could manage to handle the x509 certificate properly. cryptography is an OpenSSL dependency, anyway, so I assume it is being used as the backend. @blaggacao  do you think OpenSSL can be safely removed?

**Comment by blaggacao** [comment]
I guess so, for [this part](https://github.com/nubark/facturark/blob/510576784e7dad7c895cc3b0356f2d94566643bb/facturark/signer/signer.py#L89), see: https://cryptography.io/en/latest/hazmat/primitives/asymmetric/serialization/?highlight=pkcs12#cryptography.hazmat.primitives.serialization.pkcs12.load_key_and_certificates

a [`dump_certificate`](https://github.com/nubark/facturark/blob/510576784e7dad7c895cc3b0356f2d94566643bb/facturark/signer/signer.py#L93) replacement should just be around the corner...

That should be all, as far as I can see.

**Comment by blaggacao** [comment]
Implicitly fixed by #21
